core/storage/suggestion/gae_models.py

```diff
@@ -421,6 +421,60 @@ def get_suggestions_waiting_too_long_for_review(
             cls.last_updated
         ).fetch(MAX_NUMBER_OF_SUGGESTIONS_TO_EMAIL_ADMIN)
 
+    @classmethod
+    def get_translation_suggestions_submitted_within_given_dates(
+        cls, to_date: datetime.datetime, from_date: datetime.datetime,
+        user_id: str, language_code: str
+    ) -> Sequence[GeneralSuggestionModel]:
+        """Gets all suggestions which are are submitted within the given
+        date range.
+
+        Args:
+            to_date: Date. The date that suggestions are submitted before.
+            from_date: Date. The date that suggestions are submitted after.
+            user_id: str. The id of the user who made the submissions.
+            language_code: str. The language that the contributions should be
+                fetched.
+
+        Returns:
+            list(SuggestionModel). A list of suggestions that are submitted
+            within the given date range.
+        """
+        return cls.get_all().filter(datastore_services.all_of(
+            cls.created_on < to_date,
+            cls.created_on > from_date,
```

Comment:
Should this be <= and >=? I would expect the certificate to capture contributions on the start and end date as well.

Reply:
Done